Bai-tap-Function-2/Bai3.py

```diff
@@ -0,0 +1,17 @@
+def print_rec(m, n):
+  """
+  In ra hình chữ nhật có chiều dài m và chiều rộng n.
+
+  Args:
+    m: Chiều dài của hình chữ nhật.
+    n: Chiều rộng của hình chữ nhật.
+  """
+  if m <= n:
+        print('Fail')
+  else:   
+    for i in range(n):
+        for j in range(m):
+         print('*', end='')
+        print() 
+
+print_rec(2, 4)
```

Comment:
@ngochan21098 Please add a new line.

Reply:
@ddhienpy I have fixed it here 
https://github.com/ngochan21098/learning-python-basic/pull/1/commits/4f0786b56802b21ec8c79670e2c95a79be05bde1